### ISSUE TITLE
bowel irritability

### ISSUE THREAD

**Issue by ahamosh** [obophenotype/human-phenotype-ontology#6662]
**Preferred term label:**

**Synonyms**
irritable bowel

**Definition (free text, please give PubMed ID)**
Intermittent abdominal pain with diarrhea and/or constipation

**Parent term (use hpo.jax.org/app)**
Abnormal large intestine physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Thanks!
I think the term fits better here
https://hpo.jax.org/app/browse/term/HP:0011458

I added the following nano-attribution: OMIM:ahamosh
